ProcessOptimizer/model_systems/model_system.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from ProcessOptimizer import expected_minimum
+
+class ModelSystem:
+    """
+    Model System for testing the ProcessOptimizer. Instances of this class 
+    will be used in the example notebooks. 
+
+    Parameters:
+    * `score` [callable]:
+        Function for calculating the score of the system at a given point in 
+        the parameter space. It is expected to have the following signature:
+
+            def score(x, rng=np.random.default_rng(), noise_std=noise_value):
+                # * x is the point in the parameter space where the score will 
```

Comment:
It is somewhat unclear here if the user should supply x in the coordinates of the parameter space or in a normalized parameter space. Does it make any difference?